kong/plugins/log-serializers/alf.lua

```diff
@@ -57,6 +59,7 @@ local _M = {}
 -- ngx_http_core_module when possible.
 -- Public for unit testing.
 function _M.serialize_entry(ngx)
+
```

Comment:
Added a line, then deleted it and forgot to remove the space. I'll amend the commit.